src/Core/Utilities/StrictEmailAddressAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace Bit.Core.Utilities
+{
+    public class StrictEmailAddressAttribute : ValidationAttribute
+    {
+        public StrictEmailAddressAttribute()
+            : base("The {0} field is not a valid e-mail address.")
+        {}
+        
+        public override bool IsValid(object value)
+        {
+            var emailAddress = value?.ToString();
```

Comment:
At this point, you would then be able to call the base is valid method before continuing, i.e.:

```C#
if (!base.IsValid(value))
{
    return false;
}
```